### PR TITLE
Issue #10: [BGP] 'sh run' output contains superfluous entries

### PR DESCRIPTION
**Pull request by AKochin**
MERAprojects/ops-build#10

Change-Id: I044b0e25c0785d0227fe9756c6cc9e45d968529f
Signed-off-by: Andrey Kochin <akochin@mera.ru>

### CODE REVIEW

**Comment by ekgrish** [comment]
No questions, ok for me

File: vtysh/vtysh_ovsdb_router_context.c [hunk 1]
```diff
@@ -48,147 +218,91 @@
...
-                                      nbr_table->key_aspath_filters[i]);
-                i++;
+          /* Store pointer to peer-group ref of neighbor for quick access. */
+            if (nbr_table->bgp_peer_group) {
+                for (k = 0; k < n_pgroups; k++) {
+                    if (nbr_table->
+                        bgp_peer_group == pgroup_tables_ptr[k]) {
+                            pgroup_ptr = pgroup_tables_ptr[k];
+                    }
+                }
```
Comment (ekgrish) [inline vtysh/vtysh_ovsdb_router_context.c]: Can we avoid using of extraarray "pgroup_tables_ptr" at all, if condition (267th line) would be like:
if (nbr_table->bgp_peer_group) {
pgroup_ptr = nbr_table->bgp_peer_group;
} else {
pgroup_ptr = NULL;
}
As I understand we have the same result.

**Comment by AKochin** [comment]
Applied comments and removed unnecessary part of code.

**Review by ekgrish — COMMENTED** [review]
No more questions. All is ok.

**Review by stroykov — APPROVED** [review]

**Pull request merged**